nestor_api/lib/git.py

```diff
@@ -0,0 +1,119 @@
+"""git library"""
+
+import semver
+
+import nestor_api.lib.config as config
+import nestor_api.lib.io as io
+from nestor_api.utils.logger import Logger
+
+
+def branch(repository_dir, branch_name):
+    """Checkout a branch of a repository"""
+    Logger.debug({"path": repository_dir}, "[git#branch] Repository path")
+
+    stdout = io.execute(f"echo $(git branch --list {branch_name})", repository_dir)
```

Comment:
Do you need the `echo` ?
```suggestion
    stdout = io.execute(f"git branch --list {branch_name}", repository_dir)
```

Reply:
Indeed, it is not useless, I removed it 👍